README.md

```diff
@@ -513,6 +627,10 @@ The environment banner is a visual indication of the environment you are current
 
 You can enable/disable the banner using the following methods: `enableEnvironmentBanner` and `disableEnvironmentBanner`.
 
+#### rdfa-awareness
+
+The serialization format of the comment blocks once again uses rdfa.
```

Comment:
```suggestion
The serialization format of the comment blocks uses rdfa.
```

Documentation/readme isn't always read in order, so it can be hard to know what once again refers too. (could also refer directly to rdfa-variables e.g. `.... just like rdfa-variables`, but I don't see a direct reason for this).